packages/handlers/fhir/src/index.ts

```diff
@@ -4,7 +4,7 @@ import { join } from 'path';
 import { getCachedFetch } from '@graphql-mesh/utils';
 import urljoin from 'url-join';
 
-const fhirSchema = require('./fhir.schema.json');
+import fhirSchema from './fhir.schema.json';
```

Comment:
This is a compatbility issue with ESM, ESM doesn't support import json yet, see https://nodejs.org/api/esm.html#esm_experimental_json_modules

but most importantly, since we are using rollup, is this error:
Error: Unexpected token (Note that you need @rollup/plugin-json to import JSON files)

currently bob doesn't support adding custom plugins